R/sort_ard_hierarchical.R

```diff
@@ -184,7 +192,8 @@ sort_ard_hierarchical <- function(x, sort = c("descending", "alphanumeric")) {
         dat |>
           dplyr::mutate(
             group1 = "..empty..",
-            variable = NA
+            group1_level = list(NA),
+            variable = NA,
```

Comment:
Same here: can you help me understand what this change does?

Reply:
This is so that when `unnest()` is applied, `total_n` and `attribute` rows are not removed and are sorted last.

Reply:
thank you for the clarification!